packages/manager/modules/vps/src/dashboard/tile/configuration/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,14 @@
+{
+  "vps_dashboard_tile_configuration_title": "Votre configuration",
+  "vps_dashboard_tile_configuration_vcore_upscale": "Ajouter des vCores en passant à la gamme supérieure",
```

Comment:
"Modèle" is missing:
```
"vps_dashbaord_tile_configuration_model": "Modèle",
```

Reply:
"Modèle" exists already in https://github.com/ovh/manager/blob/master/packages/manager/modules/vps/src/translations/Messages_fr_FR.json#L449. For the moment, I've only put the new translations. We can maybe move the existing one in this folder after, what do you think ?